lib/charms/loki_k8s/v1/loki_push_api.py

```diff
@@ -534,12 +534,17 @@ def _alert_rules_error(self, event):
 # To update Promtail version you only need to change the PROMTAIL_VERSION and
 # update all sha256 sums in PROMTAIL_BINARIES. To support a new architecture
 # you only need to add a new key value pair for the architecture in PROMTAIL_BINARIES.
-PROMTAIL_VERSION = "v2.5.0"
+PROMTAIL_VERSION = "v2.9.7"
 PROMTAIL_BINARIES = {
     "amd64": {
         "filename": "promtail-static-amd64",
-        "zipsha": "543e333b0184e14015a42c3c9e9e66d2464aaa66eca48b29e185a6a18f67ab6d",
-        "binsha": "17e2e271e65f793a9fbe81eab887b941e9d680abe82d5a0602888c50f5e0cac9",
+        "zipsha": "6873cbdabf23062aeefed6de5f00ff382710332af3ab90a48c253ea17e08f465",
+        "binsha": "28da9b99f81296fe297831f3bc9d92aea43b4a92826b8ff04ba433b8cb92fb50",
+    },
+    "arm64": {
```

Comment:
How is this picked up? Note that sometimes arm64 is coded as aarch64.
We have code somewhere (gaganet?) that handles it.

Reply:
Good point, I've added that code section you mentioned (which was in cos-proxy-operator) here:

```python
if arch in ["x86_64", "amd64"]:
    self._arch = "amd64"
elif arch in ["aarch64", "arm64", "armv8b", "armv8l"]:
    self._arch = "arm64"
else:
    self._arch = arch
```